### DISCUSSION TITLE
API Media URL

### DISCUSSION THREAD

**Discussion by j-scheid** [strapi/strapi#6318] [Legacy Topics]
After I used the Digital Ocean One-Click-Install, the Strapi API responds the media URLs as something like “url:’/uploads/thumbnail.png’”. I would like to get the complete URL that links to my file as value for “url”. For example: “url:’https://example.org/uploads/thumbnail.png’”

The official strapi documentation (https://strapi.io/documentation/3.0.0-beta.x/content-api/api-endpoints.html#get-entries) shows the full URL as response as default. How can I achieve this with the one-click installation?

**Comment by derrickmehaffy** [comment]
Hi @j-scheid those full URLs come from using an upload provider such as AWS-S3, Cloudinary, ect.

Using the One-Click documentation: https://strapi.io/documentation/3.0.0-beta.x/installation/digitalocean-one-click To get access into the CLI of your one-click (likewise using the service user, and changing to the project path) you can install a provider using the following docs:  https://strapi.io/documentation/3.0.0-beta.x/plugins/upload.html#using-a-provider

**Reply by derrickmehaffy** [reply]
Currently the local provider doesn't offer full URLs

**Reply by j-scheid** [reply]
Thank you very much! :)
